content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -14,70 +14,116 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 
-// Define some important constants to initialize tracing with
-var serviceName = "MyCompany.MyProduct.MyService";
-var serviceVersion = "1.0.0";
-
 var builder = WebApplication.CreateBuilder(args);
 
-// Configure important OpenTelemetry settings, the console exporter, and instrumentation library
-builder.Services.AddOpenTelemetryTracing(tracerProviderBuilder =>
+// .. other setup
+
+builder.Services.AddOpenTelemetry()
+    .WithTracing(tracerProviderBuilder =>
+        tracerProviderBuilder
+            .AddSource(DiagnosticsConfig.ActivitySource.Name)
+            .ConfigureResource(resource => resource
+                .AddService(DiagnosticsConfig.ServiceName))
+            .AddAspNetCoreInstrumentation()
+            .AddConsoleExporter());
+
+// ... other setup
+
+public static class DiagnosticsConfig
 {
-    tracerProviderBuilder
-        .AddConsoleExporter()
-        .AddSource(serviceName)
-        .SetResourceBuilder(
-            ResourceBuilder.CreateDefault()
-                .AddService(serviceName: serviceName, serviceVersion: serviceVersion))
-        .AddHttpClientInstrumentation()
-        .AddAspNetCoreInstrumentation()
-        .AddSqlClientInstrumentation();
-});
-
-var app = builder.Build();
-
-var MyActivitySource = new ActivitySource(serviceName);
-
-app.MapGet("/hello", () =>
+    public const string ServiceName = "MyService";
+    public static ActivitySource ActivitySource = new ActivitySource(ServiceName);
+}
+```
+
+At this stage, you should be able to run you site, and see a Console output
+similar to this:
+
+<details>
+<summary>View example output</summary>
+
+```
+Activity.TraceId:            54d084eba205a7a39398df4642be8f4a
+Activity.SpanId:             aca5e39a86a17d59
+Activity.TraceFlags:         Recorded
+Activity.ActivitySourceName: Microsoft.AspNetCore
+Activity.DisplayName:        /
+Activity.Kind:               Server
+Activity.StartTime:          2023-02-21T12:19:28.2499974Z
+Activity.Duration:           00:00:00.3106744
+Activity.Tags:
+    net.host.name: localhost
+    net.host.port: 5123
+    http.method: GET
+    http.scheme: http
+    http.target: /
+    http.url: http://localhost:5123/
+    http.flavor: 1.1
+    http.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.50
+    http.status_code: 200
+Resource associated with Activity:
+    service.name: MyService
+    service.instance.id: 2c7ca153-e460-4643-b550-7c08487a4c0c
+```
+
+</details>
+
+### Manual Instrumentation
+
+Next, add [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry) via
+the `System.Diagnostics` API.
+
+Paste the following code into your `HomeController`'s `Index` action:
+
+```csharp
+public IActionResult Index()
 {
     // Track work inside of the request
-    using var activity = MyActivitySource.StartActivity("SayHello");
+    using var activity = DiagnosticsConfig.Source.StartActivity("SayHello");
```

Comment:
alternate suggestion: show http client call here, and show the HttpClient instrumentation. It might be too much for a getting-started doc, but I think its more appealing to a getting-started doc than showing manual tracing. It also potentially causes users to simply create spans in their controllers, instead of adding on to the existing span.

Reply:
I tend to agree, however, I would prefer to leave that to a larger refactor of the structure of the whole section.

Reply:
This is a manual instrumentation section, I'm not against adding a HTTP client example here too, but I don't feel we should remove manual instrumentation. Happy to add the http client example in another PR.